.eslintrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
We should really just make this the standard, I have to put this in every project I touch and it's a necessity to write efficient code.